src/components/GameView/ScrapDialog.vue

```diff
@@ -1,47 +1,36 @@
 <template>
-  <v-dialog v-model="show" max-width="650" scrollable>
+  <base-dialog v-model="show" title="Scrap Pile" id="scrap-pile-dialog">
```

Comment:
The tests specifically look for an element with id="scrap-dialog", so the name change is causing the current test failure. It's simplest to just keep the id the same as it was (if we had to change it though, we would also need to change the tests to look for this new id)
```suggestion
  <base-dialog v-model="show" title="Scrap Pile" id="scrap-dialog">
```